src/types/error.rs

```diff
@@ -44,7 +45,23 @@ impl JsError {
         })
     }
 
+    #[cfg(feature = "napi-runtime")]
+    pub fn error<'a, C: Context<'a>, S: AsRef<str>>(cx: &mut C, msg: S) -> NeonResult<Handle<'a, JsError>> {
+        let (ptr, len) = if let Some(small) = Utf8::from(msg.as_ref()).into_small() {
```

Comment:
Are we able to re-use the `JsString` code for this?

Reply:
JsString will create a struct and then i have to use to_raw on it is not ideal according to me